### DISCUSSION TITLE
questions re. long-term use

### DISCUSSION THREAD

**Discussion by timothoms** [teableio/teable#898] [Q&A] [answered]
I have extensive experience working with Airtable (by necessity rather than choice), and find it limiting, slow, and much too expensive for large projects. After some experimentation, Teable looks very promising with regard to user friendliness, speed, and most importantly, control over the data. Congratulations on building great software!

However, I am hesitant to start using Teable for serious projects because it is entirely unclear from the website and documentation how it will be commercialized and what that means for long-term use. (For context, my use case is not commercial but large collaborative academic and policy research data projects.)

So I have several questions:

- When will a subscription model be coming to Teable?

- Will it be simple to upgrade from Community Edition to Enterprise Edition with already existing bases, or downgrade?

- Will it always be possible to self-host Teable after it is out of beta release? If so, will self-hosting only be available for the Community Edition?

- Will the Community Edition remain open source?

**Comment by tea-artist** [comment, marked as answer]
Thank you for your questions about Teable. I'm happy to provide some clarity on our future plans and commercial model:

1. **Yes**, we plan to introduce subscription options in the future, including both Enterprise Edition self-hosted subscriptions and online subscriptions.

2. Upgrading from Community to Enterprise Edition (or vice versa) with existing bases **will be straightforward**. Data compatibility between Community and Enterprise Editions will be maintained, allowing easy switching within the same version. You can also choose to use the Enterprise Edition image, even without activation, to access all Community Edition features.

3. **Self-hosting will always be supported**, even after the beta release. It will be available for both the Community Edition and the Enterprise Edition. In fact, self-hosting the Enterprise Edition is our primary business model.

4. **Absolutely**, the Community Edition will always remain open-source and free. This separation of editions ensures that the open-source nature of the Community Edition can continue healthily and sustainably.

We're committed to providing flexibility and long-term stability for our users, whether they choose the Community or Enterprise Edition. Let me know if you have any further questions!
